smac/optimizer/ei_optimization.py

```diff
@@ -675,6 +675,111 @@ def _maximize(
         return next_configs_by_acq_value
 
 
+class LocalAndSortedPriorRandomSearch(AcquisitionFunctionMaximizer):
+    """Implements SMAC's default acquisition function optimization.
+
+    This optimizer performs local search from the previous best points
```

Comment:
Maybe with @hvarfner 's comment about what's going on

> There is a new local search added, which retrieves half of the starting points from the prior and half uniformly. This was added strictly because it proved to be better empirically. If we just go with the default, the local search starting points will be sampled from the prior, which is a bit too biased (particularily if the prior is incorrect). So, the new local search needs to sample from two different configspaces - the one defined by the user and one where all parameters are uniform.